spec/requests/oid_minter_request_spec.rb

```diff
@@ -2,69 +2,93 @@
 require 'rails_helper'
 
 RSpec.describe "Request new OIDs", type: :request do
-  context "Requesting a single OID" do
-    it "returns a successful JSON response" do
-      headers = { 'ACCEPT' => "application/json" }
-      get new_oid_path, headers: headers
-      expect(response).to be_successful
-      expect(response.content_type).to eq "application/json; charset=utf-8"
-      json_response = JSON.parse(response.body)
-      expect(json_response['oids']).not_to be_nil
-      oids = json_response['oids']
-      expect(oids).to be_a Array
-      expect(oids.length).to be 1
-      expect(oids[0]).to be_a Integer
+  let(:user) { FactoryBot.create(:user) }
+  describe "when authenticated" do
+    context "Requesting a single OID" do
+      it "returns a successful JSON response" do
+        headers = login_header(user).merge('ACCEPT' => "application/json")
```

Comment:
I wasn't familiar with this `login_header(user)` method, that seems super handy.

Reply:
It's not a Rails built in helper method - just a custom method we created in spec/support/auth_helper.rb to handle the authentication login.